src/OrchardCore.Modules/OrchardCore.Flows/Models/FlowPart.cs

```diff
@@ -7,4 +8,8 @@ public class FlowPart : ContentPart
 {
     [BindNever]
     public List<ContentItem> Widgets { get; set; } = [];
+
+    [BindNever]
+    [JsonIgnore]
```

Comment:
Why?

Reply:
That was not the right way to do it. I've fixt it in #17739.